### PR TITLE
[BXMSPROD-1845] elasticsearch removed from appformer

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1845

Related PRs:

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2097
* https://github.com/kiegroup/appformer/pull/1312
* https://github.com/kiegroup/kie-wb-common/pull/3781
* https://github.com/kiegroup/drools-wb/pull/1558
* https://github.com/kiegroup/jbpm-wb/pull/1572
* https://github.com/kiegroup/optaplanner-wb/pull/413
* https://github.com/kiegroup/kie-wb-distributions/pull/1196

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -161,8 +161,8 @@
     <version.org.assertj>3.14.0</version.org.assertj>
     <version.org.bouncycastle>1.68</version.org.bouncycastle>
     <version.org.codehaus.btm>2.1.4</version.org.codehaus.btm>
-    <version.org.codehaus.groovy>2.0.5</version.org.codehaus.groovy>
     <version.org.codehaus.jackson>1.9.13</version.org.codehaus.jackson>
+    <version.org.codehaus.groovy>2.0.5</version.org.codehaus.groovy>
```
Comment (mareknovotny) [inline pom.xml:165]: needed change?

**Review by lampajr — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
we also need someone from QE to check the result and possible issues from removed dependencies.
@tomasdavidorg do you think that you or somebody else can look?

**Comment by tomasdavidorg** [comment]
I will take a look.

**Comment by Ginxo** [comment]
Unstable result from FDB:
* droolsjbpm-integration: testPersistenceSubProcessWithListeners 
`[testPersistenceSubProcessWithListeners](https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/main/job/fdb/job/droolsjbpm-build-bootstrap-main.fdb/565/testReport/org.kie.spring.tests.persistence/JPASingleSessionCommandServiceFactoryTest/testPersistenceSubProcessWithListeners/)`
* kie-wb-common: annotationsWithMemberOfTypeClass being discussed at https://github.com/kiegroup/kie-wb-common/pull/3781

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me. I didn't see any problem after the removal.

**Review by adrielparedes — APPROVED** [review]

**Comment by mareknovotny** [comment]
FDB failures are looking as flaky tests.

**Comment by mareknovotny** [comment]
i found out few remaining items, please check them out too:

```
find . -type f -name pom.xml -exec grep elasticsearch {} \; -print
        <artifactId>jbpm-event-emitters-elasticsearch</artifactId>
        <artifactId>jbpm-event-emitters-elasticsearch</artifactId>
./jbpm-bom/pom.xml
        <artifactId>kie-soup-dataset-elasticsearch</artifactId>
        <artifactId>kie-soup-dataset-elasticsearch</artifactId>
./kie-soup-bom/pom.xml
        <artifactId>uberfire-metadata-backend-elasticsearch</artifactId>
        <artifactId>uberfire-metadata-backend-elasticsearch</artifactId>
./uberfire-bom/pom.xml
                        <groupId>org.elasticsearch</groupId>
                        <artifactId>elasticsearch</artifactId>
                               See https://discuss.elastic.co/t/an-unshaded-joda-time-classes-in-elasticsearch-2-1-1-jar/38943
./pom.xml
```

**Pull request merged**